scripts/persistent-docker-setup/setup.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+set -e 
+
+echo "Starting node environment"
+
+configure_network() {
```

Comment:
Since this is all setup and used by the docker file interally, this and the detect_ips can be one single script no?

Reply:
Yeah, but I was thinking that it would `setup.sh` more complex than it needs to be.